hack/kube/overlays/dev/preprocessing-moma-secret.yaml

```diff
@@ -18,3 +18,5 @@ stringData:
 
     [worker]
     maxConcurrentSessions = 1
+  .succumb: |
```

Comment:
I don't understand why you've named this file ".succumb".  How is the name relevant to the functionality?

Reply:
I named it succumb because the files 'succumb' to deletion/removal and its the opposite of endure. So tongue in cheek, these are the opposition of files that endure(enduro) so that is why it is a succumb file.

Reply:
I think it would be better to use a name that makes clear the intent of the file, like ".remove", ".forbidden" or ".prohibit".

Reply:
I think the name succumb is a bit more fun and clear enough but I will change it.